styles/Colors.js

```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+
+export const Colors = {
+  Purple: '#863AE8',
+  Darkgrey: '#bcbcbc',
+  Lightgrey: '#fafafa',
+  Glack: '#151515'
```

Comment:
Glack!

Reply:
I'm withdrawing this. I figured this out when i'm doing font style haha